### ISSUE TITLE
Shows too many lines in Gnome Settings

### ISSUE THREAD

**Issue by BrunoVernay** [rafaelmardojai/blanket#82]
In Gnome Settings there is one line per Blanket sound.
There should be only one general volume for Blanket here.
![image](https://user-images.githubusercontent.com/132261/103081895-3c7d5a80-45d9-11eb-8061-b7fd54d2234c.png)

Using Gnome 3.38.2 on Fedora 33

**Comment by rafaelmardojai** [comment]
I can't do anything, for example web browsers also show one line per video/audio playing. I don't know if GStreamer allows me to merge different sounds into one player. 

Maybe GNOME could group volume bars per app?

**Comment by BrunoVernay** [comment]
I guess it should be possible to mixdown to stereo inside Blancket to present only a stereo output to the outside.

I use many sounds together, that is why this jumped at me, but it is a minor issue.  Feel free to postpone to next year :-)

**Comment by SeaDve** [comment]
> I can't do anything, for example web browsers also show one line per video/audio playing. I don't know if GStreamer allows me to merge different sounds into one player.

You could possibly create a custom pipeline. You can use audiomixer to join multiple audiostreams, this could also possibly make it more efficient, since you are essentially just playing with a single pipeline/player.

**Comment by rafaelmardojai** [comment]
> You could possibly create a custom pipeline. You can use audiomixer (a gst element) to join multiple audiostreams, this could also possibly make it more efficient, since you are essentially just playing with a single pipeline/player.

Yep, I checked audiomixer recently, but didn't have time to try. Soon I will have time to work on some Blanket issues.

**Comment by C-512L** [comment]
@rafaelmardojai Friendly ping. Has there been any progress on #313?

**Comment by rafaelmardojai** [comment]
@C-512L Right now is a a bit blocked because I have found some difficulties to correctly implement looping with audio mixing. Right now the implementation of that PR works but has performance issues, I have do have some fixes for that locally but looping has bugs.

So I probably need some time to understand GStreamer more deeply to be able to push this further, but right now I lack that time.
